mlkem/params.h

```diff
@@ -18,7 +18,7 @@
 #define MLKEM_N 256
 #define MLKEM_Q 3329
 #define MLKEM_Q_HALF ((MLKEM_Q + 1) / 2) /* 1665 */
-#define UINT12_LIMIT 4096
+#define MLK_UINT12_LIMIT 4096
```

Comment:
Do you want to change this to MLKEM_UINT12_LIMIT? I know this is hardly a scheme "parameter" (and, hence, should not be named MLKEM_ or be placed in params.h), but I don't know where else it should go and having a single `MLK_` define in this file looks weird.

Reply:
Ok